app/models/answer.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Answer < ApplicationRecord
+    validates :title, :is_correct, presence: true
+
+    belongs_to :question
+    has_many :player_answers
```

Comment:
You should also add a `dependent: :destroy` option to your `has_many` relationship. 

As per [Rails' Documentation](https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent) this option will ensure that if you destroy an `Answer`, it will also destroy all its `PlayerAnswers` first - so that you don't have to manually do it yourself to ensure the foreign key consistency.